server/routes.py

```
@@ -13,7 +13,7 @@ def index():
 
     if name:
         cursor.execute(
-            "SELECT * FROM books WHERE name LIKE %s", name
+            "SELECT * FROM books WHERE name LIKE '%" + name + "%'"
```

Ruff (0.14.13)

Possible SQL injection vector through string-based query construction

(S608)